odoo/tests/common.py

```diff
@@ -558,17 +572,18 @@ def _chrome_start(self):
             '--remote-debugging-address': HOST,
             '--remote-debugging-port': str(self.devtools_port),
             '--no-sandbox': '',
+            '--disable-crash-reporter': '',
```

Comment:
Does this flag also exists in Chromium?

Reply:
Yep [here](https://cs.chromium.org/chromium/src/headless/app/headless_shell_switches.cc?q=kDisableCrashReporter&sq=package:chromium&type=cs&l=32)